src/test/resources/syntheticobjectcreation/expected/org/can/not/be/real/Pen.java

```diff
@@ -0,0 +1,7 @@
+package org.can.not.be.real;
+public class Pen {
+
+    public  Pen(org.still.not.real.Ink parameter0) {
```

Comment:
Is it possible to fix the way that we print these so that there is not an awkward double-space before the name of the constructor?

Reply:
I think we can. I will try that.